authors/apps/authentication/models.py

```diff
@@ -126,16 +128,121 @@ def get_short_name(self):
         """
         return self.username
 
-    def _generate_jwt_token(self):
+    def _generate_jwt_token(self, days):
         """
         Generates a JSON Web Token that stores this user's ID and has an expiry
         date set to 60 days into the future.
         """
-        dt = datetime.now() + timedelta(days=60)
+        dt = datetime.now() + timedelta(days)
```

Comment:
Can we leave this with the `60 days` constraint?

Reply:
The token hashing is used by two features. One for authentication is set to 60 days whereas the one for password reset I had to limit it to a day. Thought it wise that a reset token shouldn't last as long. Happy to be advised differently though.

Reply:
This approach is okay and yes, it is better for a reset token to not last as long.